src/components/Text/Text.js

```diff
@@ -23,9 +24,42 @@ export default class Text extends React.Component {
         break;
     }
 
+    switch (this.props.lineHeight) {
```

Comment:
Is this necessary for adding the `default`option?

Reply:
removed switch case